powerline/lemonbar.py

```diff
@@ -0,0 +1,23 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+from powerline import Powerline
+from powerline.lib.dict import mergedicts
+
+
+class LemonbarPowerline(Powerline):
+	def init(self, output=None):
```

Comment:
This method should be missing output argument, as well as `self.output = output`.